lib/matplotlib/colors.py

```diff
@@ -529,7 +529,18 @@ def __call__(self, X, alpha=None, bytes=False):
         if vtype == 'scalar':
             rgba = tuple(rgba[0, :])
         return rgba
-
+    
+    def __copy__(self):
+        """Create new object with the same class and update attributes
+	If object is initialized, copy the elements of _lut list
+	"""
+	cls = self.__class__
```

Comment:
Is this really the best way to do this?  It seems to be dropping down to a low layer of python.